guide/russian/angularjs/index.md

```diff
@@ -26,5 +27,3 @@ HTML:
 ```
 
 HTML
-
-\`\` \`
```

Comment:
@cmccormack I think this is supposed to somehow tie to line 14? It isn't markdown syntax that I have seen before.

Reply:
@1andee you are correct, I don't know what the use of either line is but I reverted my commit.